compact.go

```diff
@@ -54,7 +54,7 @@ type Compactor interface {
 	// Results returned when compactions are in progress are undefined.
 	Plan(dir string) ([]string, error)
 
-	// Write persists a Block into a directory.
+	// Write persists index and chunks from a Block for samples overlaps with mint:maxt range into a directory.
```

Comment:
Because saying "new block is the union of given blocks" is not true. It is the union of new blocks that intersect with request min and max time, right? And that's why I changed this comment and others for `populate` as well.

Reply:
yep that is right. 
How about:
`// Write persists a Block into a directory by including data within the requested mint:maxt range.`

Reply:
ok , thanks
Done.